node/web/src/lib.rs

```diff
@@ -98,7 +105,8 @@ pub async fn run(
 
 async fn setup_node(
     block_producer: Option<AccountSecretKey>,
-    seed_nodes_url: Option<String>,
+    seed_nodes_urls: Option<Vec<String>>,
+    seed_nodes_fixed: Option<Vec<String>>,
```

Comment:
Nit: Can we have a small comment describing the different between `seed_nodes_urls` and `seed_nodes_fixed`, please? We can infer it by looking at the code, but as the types are both the same, it is always a good practice to have a small line of docs.